### ISSUE TITLE
Does amd smi exporter support mi250 GPUs?

### ISSUE THREAD

**Issue by yx-lamini** [amd/amd_smi_exporter#13]
This says only MI300x. Just to confirm.
![image](https://github.com/user-attachments/assets/f1edbb3f-4cf8-4240-b87d-77b8c6a463f4)

**Comment by muralimk-amd** [comment]
Yes This is supporting on MI250 also. will update the family details in README.
Thanks for pointing out.

**Comment by muthusAMD** [comment]
Hi @yx-lamini ,
We have updated the [Readme ](https://github.com/amd/amd_smi_exporter/blob/master/README.md)with MI200 Support,
![Readm_MI200Support](https://github.com/user-attachments/assets/bafd594a-9cce-45ed-8a58-da52f23d24d0)
Thanks for pointing out :)
